includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -915,6 +917,23 @@ private static function settings_appearance() {
 
 		WPBDP_Admin_Education::add_tip_in_settings( 'table', 'themes' );
 
+        if ( ! is_plugin_active( 'font-awesome/index.php' ) || ( ! class_exists( '\FortAwesome\FontAwesome_Loader' ) ) ) {
+			wpbdp_register_setting(
+				array(
+					'id'    => 'fontawesome-enabled',
+					'name'  => _x( 'FontAwesome styles?', 'settings', 'business-directory-plugin' ),
```

Comment:
Let's change this to `__( 'Font Awesome', 'business-directory-plugin' ),` (It's two words)